corehq/apps/geospatial/reports.py

```diff
@@ -88,6 +89,19 @@ def _get_geo_location(self, case):
     def rows(self):
         raise NotImplementedError()
 
+    @property
+    def disbursement_parameters(self):
+        try:
+            config = GeoConfig.objects.get(domain=self.domain)
+        except GeoConfig.DoesNotExist:
+            return []
+
+        unspecified = _("Unspecified")
```

Comment:
nit: Would be nice to keep this logic in view. We are now assigning the value to "Unspecified" which is confusing since that should never be the value for the min/max cases per user.

Reply:
@mkangia The max value criteria can be unspecified (by the user), which means the algorithm figures out what the value should be on the fly. Or do you mean we should show the values with which the algorithm actually runs?

Reply:
Which would not be a bad idea.

Reply:
i was trying to bring attention to the assignment of "Unspecified" to the value which was confusing since value would be an integer. So i suggested that we display that if needed as a part of HTML/JS instead of putting "Unspecified" as the value in django view itself.

Reply:
Oh, I see. That's a good point.